src/net/sourceforge/kolmafia/session/EquipmentManager.java

```diff
@@ -667,6 +667,15 @@ public static final void removeConditionalSkills(final Slot slot, AdventureResul
         KoLCharacter.removeAvailableSkill(SkillPool.SWOOP_LIKE_A_BAT);
         KoLCharacter.removeAvailableSkill(SkillPool.SUMMON_CAULDRON_OF_BATS);
       }
+      case ItemPool.SHERIFF_BADGE -> {
+        KoLCharacter.removeAvailableSkill(SkillPool.ASSERT_YOUR_AUTHORITY);
```

Comment:
you want an "add available skill" on each if the others are equipped. Like we do for bear arms.

Reply:
also you can merge these cases because they're identical